### DISCUSSION TITLE
Filament PHP Form Component

### DISCUSSION THREAD

**Discussion by emjay64** [Lakshan-Madushanka/laravel-comments#65] [Q&A] [answered]
Is there a simple way to add the comments to filament php resource form?

**Comment by Lakshan-Madushanka** [comment, marked as answer]
Follow the same general steps. Ensure Step [3](https://lakm.gitbook.io/commenter/basics/installation#step-3) is implemented as described in the https://filamentphp.com/docs/3.x/support/assets#registering-css-files and [Filament documentation for registering JavaScript files](https://filamentphp.com/docs/3.x/support/assets#registering-javascript-files) respectively.

Instead of using directives (`@commentsScripts` and `@commentsStyles`), reference the resource paths located in the `public\vendor\lakm\laravel-comments\build` directory.
